src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheck.java

```diff
@@ -760,17 +757,7 @@ private static int compareImports(String import1, String import2) {
      * @return count of empty lines before given.
      */
     private int getCountOfEmptyLinesBefore(int lineNo) {
-        int result = 0;
-        final String[] lines = getLines();
-        //  [lineNo - 2] is the number of the previous line
-        //  because the numbering starts from zero.
-        int lineBeforeIndex = lineNo - 2;
-        while (lineBeforeIndex >= 0
-                && CommonUtil.isBlank(lines[lineBeforeIndex])) {
-            lineBeforeIndex--;
-            result++;
-        }
-        return result;
+        return CommonUtil.getCountOfEmptyLinesBefore(lineNo, getLines());
```

Comment:
Due to a edge case that getCountOfEmptyLinesBefore() could not pass jacoco code coverage, the logic is split into CommonUti.